### PR TITLE
CDF-6265: add support for labels endpoint and assets labeling

### PR DESCRIPTION
**Pull request by vincent-cognite**
For the integration tests, we will need to update the api key to have labels rw capabilities.
Is that something you can do @sanderland ?

### CODE REVIEW

**Comment by sanderland** [comment]
> For the integration tests, we will need to update the api key to have labels rw capabilities.
> Is that something you can do @sanderland ?

done

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ Changes are grouped as follows
 
 ## [0.6.0] - 2020-05-08
 
+### Added
```
Comment (sanderland) [inline CHANGELOG.md:20]: bump version please. 0.7?

File: cognite/experimental/data_classes/assets.py [hunk 2]
```diff
@@ -379,11 +381,11 @@ def __init__(
         root: bool = None,
         external_id_prefix: str = None,
         types: List[Dict[str, Any]] = None,
+        labels: List[List[Dict[str, Any]]] = None,
         cognite_client=None,
     ):
         self.name = name
         self.parent_ids = parent_ids
-        self.parent_external_ids = parent_external_ids
```
Comment (sanderland) [inline cognite/experimental/data_classes/assets.py:386]: any reason this is missing in playground?

File: cognite/experimental/_api/labels.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        return self.__call__()
+
+    def __call__(
+        self,
+        name: str = None,
+        external_id_prefix: str = None,
+        limit: int = None,
+        chunk_size: int = None,
+    ):
+        filter = LabelDefinitionFilter(name=name, external_id_prefix=external_id_prefix).dump(
```
Comment (sanderland) [inline cognite/experimental/_api/labels.py:28]: LabelFilter

File: cognite/experimental/data_classes/assets.py (hunk 2 above)
Comment (vincent-cognite) [reply cognite/experimental/data_classes/assets.py:386]: not that I know of, but it is not there at the moment, so this is just syncing that fact

**Pull request merged**